tests/promo-code/promo-code.mutation.test.js

```diff
@@ -1,32 +1,54 @@
-const { newPromoCode } = require('./promo-code.variables');
+const {
+  newPromoCode,
+  newPromoCodeForUpdate,
+} = require('./promo-code.variables');
 const { setupApp } = require('../helper-functions');
-
-const { addPromoCode, deletePromoCode } = require('./promo-code.helper');
+const {
+  addPromoCode,
+  deletePromoCode,
+  updatePromoCode,
+} = require('./promo-code.helper');
 
 let promoCodes;
 let promoCodeId;
 let operations;
 
-describe('PromoCodes queries', () => {
-  beforeAll(async () => {
+describe('PromoCodes mutations', () => {
+  beforeEach(async () => {
     operations = await setupApp();
+    promoCodes = await addPromoCode(newPromoCode, operations);
+    promoCodeId = promoCodes._id;
   });
 
-  it('should add add to database', async () => {
+  it('should add to database', async () => {
     promoCodes = await addPromoCode(newPromoCode, operations);
 
     expect(promoCodes).toHaveProperty('code', newPromoCode.code);
-    expect(promoCodes.dateFrom).toEqual('2021-12-26');
+    expect(promoCodes.dateFrom).toEqual(newPromoCode.dateFrom);
     expect(promoCodes).toHaveProperty('discount', newPromoCode.discount);
-    expect(promoCodes.discount).toEqual(12);
+    expect(promoCodes.discount).toEqual(newPromoCode.discount);
```

Comment:
why do you need a lot of expects for this? just one common

Reply:
Fixed